Client/Telemetry/UnifiedTelemetry.swift

```diff
@@ -28,5 +30,10 @@ class UnifiedTelemetry {
 
         Telemetry.default.add(pingBuilderType: CorePingBuilder.self)
     }
+
+    @objc func uploadError(notification: NSNotification) {
+        guard let error = notification.userInfo?["error"] as? NSError else { return }
+        SentryIntegration.shared.send(message: error.localizedDescription, tag: "UnifiedTelemetry", severity: .info)
```

Comment:
Can we wrap this with a check to only log this in Beta? This could be extremely high volume in production if something goes wrong with the telemetry service. Either check here or in SentryIntegration.send() (i think there already is a check to make sure Storage and SwiftData is only logged on Beta.

Reply:
Thanks for checking this, yes will add it